azure_functions_worker/testutils.py

```diff
@@ -640,6 +640,15 @@ def popen_webhost(*, stdout, stderr, script_root=FUNCS_PATH, port=None):
         if servicebus:
             extra_env['AzureWebJobsServiceBusConnectionString'] = servicebus
 
+        eventgrid_topic_uri = testconfig['azure'].get('eventgrid_topic_uri')
```

Comment:
May want to update the wiki->contributor guide, step 7 for this newly introduced config **eventgrid_topic_uri** and **eventgrid_topic_key**

Reply:
Sure. I'll do that post-merge.